Couldn't open joystick force feedback!

Trying to run the joy_node with logitech joystick controller on Kinetic version Ubuntu 16.04, but it's giving me an error [ERROR] [1561675023.851541705]: Couldn't open joystick force feedback!
Tried with multiple joy sticks and i can't figure out what's causing this and how to fix it.

--- a/joy_linux/src/joy_linux_node.cpp
+++ b/joy_linux/src/joy_linux_node.cpp
@@ -346,7 +346,7 @@ public:
         ie.value = 0xFFFFUL * gain / 100;
 
         if (write(ff_fd_, &ie, sizeof(ie)) == -1) {
-          RCLCPP_ERROR(
+          RCLCPP_WARN(
             node_->get_logger(), "Couldn't open joystick force feedback: %s", strerror(errno));
         }
 

